elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -2300,10 +2293,80 @@ module load anaconda</code>
             </list>
         </body>
     </sub-article>
-    <!-- article-type changed from reply to author-comment -->
-    <sub-article article-type="author-comment" id="sa2">
+    <sub-article id="sa2" article-type="referee-report">
         <front-stub>
             <article-id pub-id-type="doi">10.7554/eLife.00666.sa2</article-id>
+            <title-group>
+                <article-title>Reviewer recommendations</article-title>
+            </title-group>
+            <contrib-group>
+                <!-- Anonymous reviewer -->
+                <contrib contrib-type="author">
+                    <anonymous/>
+                    <role specific-use="referee">Reviewer</role>
+                </contrib>
+            </contrib-group>
+            <custom-meta-group>
+                <!-- custom-meta to indicate that this decision letter was transferred from elsewhere -->
+                <custom-meta>
+                    <meta-name>transferred-from</meta-name>
+                    <meta-value>Review Commons</meta-value>
+                </custom-meta>
```

Comment:
For review commons, I _think_ we'd probably just have a single 'peer review' `sub-article` that simply stated that the paper was reviewed at reviewed commons (like we do for the decision letter now). But I will check and confirm. Ideally we would keep this `custom-meta` though.